### ISSUE TITLE
copyq add/insert do not trigger automatic commands

### ISSUE THREAD

**Issue by Spacewalker2** [hluk/CopyQ#835]
Hello, I use copyq add a lot in my own scripts and it would be nice when this would triggers the automatic commands too. Automatic commands work fine for me when I copy anything to clipboard. I'm using copyq 3.1.2 on arch linux.

Thanks.

**Comment by amosbird** [comment]
Hi, if I remember correctly, this is by design to avoid infinite loop.

**Comment by hluk** [comment]
> if I remember correctly, this is by design to avoid infinite loop.

Correct. The side effect of this is that even running `copyq copy ...` outside automatic command will not trigger the automatic commands. The reason is also that if you have access to `copyq` from script you can tell it yourself what to do with the copied data (I usually don't want to store data I copy from script).

Is there any specific action you need the automatic commands to perform when copying from script?

**Comment by Spacewalker2** [comment]
I parse urls from rss feeds or from elsewhere but copyq is sorting them to the right tab. I use copyq a lot for storing urls and it is some kind of headquarter for this. Using copyq means I'm able to start urls in different browsers and media players.
